library/lua/gui/widgets/labels/label.lua

```diff
@@ -277,7 +277,7 @@ Label.ATTRS{
     auto_width = false,
     on_click = DEFAULT_NIL,
     on_rclick = DEFAULT_NIL,
-    scroll_keys = STANDARDSCROLL,
+    scroll_keys = Scrollbar.STANDARDSCROLL,
```

Comment:
the issue here, I think, is that widgets can't include `gui.widgets` since then it would be a circular dependency

another option is to move all constants (and settable preferences) into a separate, common file